AsyncDisplayKit/ASCellNode.h

```diff
@@ -86,17 +90,6 @@ typedef NS_ENUM(NSUInteger, ASCellNodeVisibilityEvent) {
 - (void)touchesCancelled:(nullable NSSet<UITouch *> *)touches withEvent:(nullable UIEvent *)event ASDISPLAYNODE_REQUIRES_SUPER;
 
 /**
- * Marks the node as needing layout. Convenience for use whether the view / layer is loaded or not.
- *
- * If this node was measured, calling this method triggers an internal relayout: the calculated layout is invalidated,
```

Comment:
We lost some of the detail here around re-measuring.  This description wasn't great, so we should rewrite it and yours is closer, but: it's not currently clear that -setNeedsLayout can cause the cell to /change size/, as that is not normally true for any type of UIView or UIKit cell.  Then on top of that, but only once you understand that triggering does a re-measure pass and it may result in a different size, there is the detail that the table or collection will be notified of any size change.

It might be worth mentioning that a table's cell node will be given the fixed width with a min height of 0 and Infinity to measure itself, and collections get 0, 0 and Inf, Inf (unless the constrainedSizeFor... method is overridden).  Just not sure what the best way is to emphasize these critical differences to users.

cc @Adlai-Holler, @hannahmbanana, @lappp9